src/gtk-3.0/3.22/sass/_colors.scss

```diff
@@ -21,13 +21,15 @@ $light_fg_color:  rgba($white, 0.85);
 $button_fg_color: if($variant == 'light', rgba($brown_800, 0.75), rgba($white, 0.8)); // extra fg color for raised buttons
 
 $fg_color:                    if($variant == 'light', rgba($brown_800, 0.85), $white);
+$Scrollbar_fg_color:          if($variant == 'light', rgba($grey_500, 0.95), rgba($grey_200, 0.75));
+$Scrollbar_fg_color2:         if($variant == 'light', rgba($grey_600, 0.75), rgba($grey_300, 0.85));
```

Comment:
lowercase names

Also better name would be scrollbar_active_fg_color for Scrollbar_fg_color2  to be consistent with other color variable names

Reply:
Alright, fixed the little (unnoticed :sweat_smile: ) typo.